py-polars/polars/dataframe/plotting.py

```diff
@@ -6,15 +6,12 @@
     import sys
 
     import altair as alt
-    from altair.typing import (
-        ChannelColor,
-        ChannelOrder,
-        ChannelSize,
-        ChannelTooltip,
-        ChannelX,
-        ChannelY,
-        EncodeKwds,
-    )
+    from altair.typing import ChannelColor as Color
+    from altair.typing import ChannelOrder as Order
+    from altair.typing import ChannelSize as Size
+    from altair.typing import ChannelX as X
+    from altair.typing import ChannelY as Y
+    from altair.typing import EncodeKwds
```

Comment:
Just linking the context for this change https://github.com/pola-rs/polars/pull/18609#issuecomment-2336826970

cc'ing @mattijn as this is a possible solution for https://github.com/vega/altair/issues/3582#issuecomment-2336727055